libgpg-error: Update to 1.34

neheb opened the pull request:
Remove FPIC as it is already default. Both fPIC and DPIC are passed.

Some extra configure arguments to speed up build times.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @MikePetullo 
Compile tested: ramips

diizzyy commented:
Perhaps add a few sane mirrors while at it?
https://www.gnupg.org/download/mirrors.html

```
https://ftp.bit.nl/mirror/gnupg/
http://ring.ksc.gr.jp/archives/net/gnupg/libgpg-error/ (Ring network)
https://ftp.nluug.nl/security/gnupg/ (Surfnet)
```

diizzyy commented:
@neheb 
Since they provide a mirrors and a mirror list I'm going to presume that they want to offload the main site.

neheb commented:
Added mirrors.

Simplified patch.

@hnyman can we get this in before the release? This fixes libgcrypt on armeb which is a dependency for many packages (11 from a quick git grep).
